app/templates/events/view/tickets/index.hbs

```diff
@@ -95,11 +95,11 @@
       <tr>
         {{#link-to 'events.view.tickets.orders.index' tagName='td'
         class='ui green inverted segment link'}}
-          <span>{{t 'Completed'}}</span>
```

Comment:
@geekyd right now the API doesn't support completed in the response. So I guess we can't have it here. If needed we can, later on, add it. I guess it good to merge now. 
@niranjan94 @CosmicCoder96 @sumedh123 Please review.